client/simplestreams_images.go

```diff
@@ -282,3 +287,8 @@ func (r *ProtocolSimpleStreams) GetImageAliasArchitectures(imageType string, nam
 func (r *ProtocolSimpleStreams) ExportImage(fingerprint string, image api.ImageExportPost) (Operation, error) {
 	return nil, fmt.Errorf("Exporting images is not supported by the simplestreams protocol")
 }
+
+// GetServerSupportedFilters returns two lists: one with filters supported by server and second one with not supported.
+func (r *ProtocolSimpleStreams) GetServerSupportedFilters(filters []string, i interface{}) ([]string, []string) {
+	return []string{}, filters
+}
```

Comment:
This should go away if we move to `validateFilters` or something like that instead.

Reply:
I've moved this method to `lxc/utils`.